setup.py

```diff
@@ -51,7 +51,7 @@
 
     entry_points={
         'console_scripts': [
-            'make_composite.py = asf_tools.composite:main',
+            'make_composite = asf_tools.composite:main',
```

Comment:
For this change, you also need to update:
* `tests/test_entrypoints.py`
* where it's called out in `CHANGELOG.md`